pkgs/development/python-modules/furo/default.nix

```
@@ -10,7 +10,7 @@
 
 buildPythonPackage rec {
   pname = "furo";
-  version = "2023.7.26";
+  version = "2023.8.19";
   format = "wheel";
 
   disable = pythonOlder "3.7";
```

```suggestion
  disable = pythonOlder "3.8";
```
https://github.com/pradyunsg/furo/blob/0766bb2d6d0392bf90d4e4ac447ae721fa673ee9/pyproject.toml#L17

---

Good catch!